src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -2448,6 +2448,19 @@ void mvnw(Register Rd, Register Rm,
 
 #undef INSN
 
+#define INSN(NAME, opc) \
```

Comment:
This should be in an "AdvSIMD scalar pairwise" instruction group with faddp.

Reply:
Done. The instructions `faddp`, `fmax` and `fmin` are put together in one group now.
Re-run the `test/jdk/jdk/incubator/vector/` tests and passed.